src/login/71-seat.rules.in

```diff
@@ -29,6 +29,10 @@ SUBSYSTEM=="graphics", KERNEL=="fb[0-9]", IMPORT{cmdline}="nomodeset", TAG+="mas
 SUBSYSTEM=="pci", ENV{ID_PCI_CLASS_FROM_DATABASE}=="Display controller", \
                   ENV{DRIVER}=="", IMPORT{cmdline}="nomodeset", TAG+="seat", TAG+="master-of-seat"
 
+# Synthesize a seat for graphic devices without DRM and that fall back to fb
+# device instead.
+SUBSYSTEM=="pci", ENV{ID_MASTER_OF_SEAT_FROM_DATABASE}=="1", TAG+="seat", TAG+="master-of-seat"
```

Comment:
the first check should be unnecessary. if the prop is set it should count.

I'd drop the "_FROM_DATABASE" suffix. We use that nly in case we have conceptually the same field from two sources. but here we don't hence no suffix please.